src/test/ui/consts/const_let_assign3.rs

```diff
@@ -18,6 +17,10 @@ const FOO: S = {
     s
 };
 
+type Array = [u32; { let mut x = 2; let y = &mut x; *y = 42; *y}];
```

Comment:
Could you format this on several lines so it's more clear for assorted readers what the source of the errors are?